Uchu.World/Client/CdClient/ClientCache.cs

```diff
@@ -11,6 +15,11 @@ namespace Uchu.World.Client
     /// </summary>
     public static class ClientCache
```

Comment:
Little note for the future: now that we're going to use this class instead of the CdClient entirely, the name Cache might be a bit confusing or at least not really suitable for how much it's used. In the future we could look into renaming this to something like GameData, GameInfo or LUInfo or something. You don't have to change this btw, just something to think about.

Reply:
I'll pass on the change for now as it does handle caching. If it gets more functionality, it may be better to change names.